src/i18n/ms/index.js

```diff
@@ -0,0 +1,157 @@
+import potato from './potato';
+import print from './print';
+import tester from './tester';
+export default {
+  ms: {
+    message: {
+      ...potato,
+      help: {
+        label: 'Panggil Wizard'
+      },
+      print: { ...print },
+      tester: { ...tester },
+      keyboard: {
+        label: 'papan kekunci'
+      },
+      layout: {
+        label: 'susunan'
+      },
+      keymapName: {
+        label: 'Nama Keymap',
+        placeholder: 'Sesuaikan nama keymap'
+      },
+      loadDefault: {
+        label: 'Memuat keymap asal',
+        title: 'Memuatkan keymap asal dari QMK Firmware'
+      },
+      compile: {
+        label: 'Kompil',
+        title: 'Kompil Keymap'
+      },
+      downloadKeymap: {
+        title: 'Muat turun keymap.c',
+        label: 'keymap sahaja'
+      },
+      downloadSource: {
+        title: 'Muat turun kod perisian tegar QMK',
+        label: 'Sumber Penuh'
+      },
+      downloadJSON: {
+        title: 'Eksport fail JSON QMK Keymap',
+        label: 'Keymap.JSON'
+      },
+      importJSON: {
+        title: 'Import fail JSON QMK Keymap'
+      },
+      importUrlJSON: {
+        title: 'Import fail JSON QMK Keymap dari URL'
+      },
+      printKeymap: {
+        title: 'Cetak Lapisan Keymap',
+        label: 'Cetak Keymap'
+      },
+      testKeys: {
+        title: 'Uji Input Papan kekunci',
+        label: 'Test Keyboard'
+      },
+      downloadFirmware: {
+        label: 'Perisian tegar',
+        title: 'Muat turun fail perisian tegar untuk di flash'
+      },
+      flashFirmware: {
+        label: 'Auto-Flash',
+        title: 'Flash perisian tegar yang dikompil kepada MCU secara automatik'
+      },
+      flashFile: {
+        label: 'Custom-Flash',
+        title: 'Flash file pilihan penguna kepada MCU'
+      },
+      ColorwayTip: {
+        title: 'Ctrl + Alt + N untuk pilih warna seterusnya'
+      },
+      layer: {
+        label: 'Lapisan',
+        confirm: 'Andan pasti ingin membuang lapisan?',
+        title: 'Buang Lapisan'
+      },
+      keymap: {
+        label: 'Keymap'
+      },
+      downloadToolbox: {
+        label: 'Dapatkan peralatan QMK'
```

Comment:
I think 'Dapatkan QMK Toolbox' would be better (peralatan is basically a translation which means equipment - however, since the program is called QMK Toolbox, i think it would be better to leave the name intact right? unless in other translations you translate the name too)

Reply:
Yeah. You are right.